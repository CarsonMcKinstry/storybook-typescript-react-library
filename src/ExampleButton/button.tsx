import React from 'react';
import Button from 'react-bootstrap/lib/Button';

interface IProps {
  /**
   * Text for the button
   */
  text: string;
}

export const ExampleButton = (props: IProps) => {
  return (
    <Button>
      {props.text}
    </Button>
  );
};
