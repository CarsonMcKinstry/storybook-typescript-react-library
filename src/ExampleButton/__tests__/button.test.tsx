import React from 'react';
import { shallow } from 'enzyme';
import { ExampleButton } from '../button';

it('renders without crashing', () => {
  shallow(<ExampleButton text='hey' />);
});
