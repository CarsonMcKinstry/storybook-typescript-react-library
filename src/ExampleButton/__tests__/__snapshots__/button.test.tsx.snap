// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<button
  className="btn btn-default"
  disabled={false}
  type="button"
>
  Cute button!
</button>
`;
